datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/codeowners.py

```diff
@@ -0,0 +1,74 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+
+import click
+
+from ...utils import get_codeowners, get_valid_integrations
+from ..console import CONTEXT_SETTINGS, echo_failure, echo_success
+
+DIRECTORY_REGEX = re.compile(r"\/(.*)\/$")
+
+
+IGNORE_TILES = {
```

Comment:
Can you add comments explaining why these are ignored?

Reply:
Updated!